docs/development/extensions-core/mysql.md

```diff
@@ -61,6 +61,8 @@ Depending on the MariaDB client library version, the connector supports both `jd
 
 ## Setting up MySQL
 
+To avoid issues with upgrades that require scehma changes to a large metadata table, chose a MySQL version that supports instant ADD COLUMN semantics. For example, MySQL 8.
```

Comment:
can we add a similar bit about postgres? We can just say that postgres supports instant add already but please double check anyway.

Reply:
I couldn't find when Postgres started supporting it, but for null columns it is supported for all current versions which are 11 and later, so noted that.